src/raw.js

```diff
@@ -116,12 +116,16 @@ function replaceKeyBindings(raw) {
   var client   = raw.client
   var sql      = raw.sql, bindings = []
 
-  var regex = new RegExp('(^|\\s)(\\:\\w+\\:?)', 'g')
+  var regex = new RegExp('(\\:\\w+\\:?)', 'g')
```

Comment:
Earlier bindings has been working in a way that there had to be space before start of named binding. Old regexp was: `\\s(\\:\\w+\\:?)` so I had changed it to allow also be in start of line `(^|\\s)(\\:\\w+\\:?)`. 

I don't know about original motivation why named bindings were not allowed to be used without leading white space.